BrightCoders-Proyectos/tribo-mobile
# Create TextInputCompoent and add corresponidng color styles

# Description
I created a textInput component with four states (default, active, error and disabled) with corresponding styles in order to finish issue #40 . As you can see the component uses props to change the corresponding style 
 
 Examples:
I have added textInputType props to call the corresponding style as you can see in the Screenshot section if you write t
![image](https://user-images.githubusercontent.com/69820530/104624680-78946000-5659-11eb-9284-cfa51e175f38.png) you will see the search bar.

 # Testing
1. Download repo.
2. Go to project folder (tribomobile). 
3. Switch to my branch (feature/40-text-input-component).
4. Git pull and npm install if necessary.
5. You will se welcomeGuestScreen so you have to import my component ![image](https://user-images.githubusercontent.com/69820530/104624983-d7f27000-5659-11eb-9f56-158e5b2e614b.png)
6. Then Call the component and see the 5 types. 
  ## Screenshots 
![image](https://user-images.githubusercontent.com/69820530/104643791-899d9b00-5672-11eb-8db6-ba430128d480.png)
![image](https://user-images.githubusercontent.com/69820530/104625402-58b16c00-565a-11eb-8cf1-2d29a03a04f1.png)
![image](https://user-images.githubusercontent.com/69820530/104625487-68c94b80-565a-11eb-9c53-66b261bf2e3a.png)
![image](https://user-images.githubusercontent.com/69820530/104628427-d9259c00-565d-11eb-8e2a-50a997b1f5e6.png)
